### PR TITLE
Provide more status information

### PR DESCRIPTION
**Pull request by ctron**

### CODE REVIEW

**Comment by ctron** [comment]
@enmasse-ci run tests profile=shared-iot

**Comment by ctron** [comment]
@enmasse-ci run tests profile=shared-iot

File: templates/iot/operator/020-ClusterRole-iot-operator.yaml [hunk 1]
```diff
@@ -12,5 +12,5 @@ rules:
     resources: [ "messagingusers" ]
     verbs: [ "get", "list", "watch", "update", "create", "patch", "delete" ]
   - apiGroups: [ "iot.enmasse.io" ]
-    resources: [ "iotprojects" ]
+    resources: [ "iotprojects", "iotprojects/finalizers", "iotprojects/status" ]
```
Comment (dejanb) [inline templates/iot/operator/020-ClusterRole-iot-operator.yaml]: Should we have iot-operator cluster role in the csv? I don't see any?

Comment (ctron) [reply templates/iot/operator/020-ClusterRole-iot-operator.yaml]: The cluster role is here: https://github.com/EnMasseProject/enmasse/pull/3179/files#diff-6b8bf1a44159704afaf341f840ea0025R516

But it was missing the sub-resources. Fixed it.

**Pull request merged**